certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3443 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.4",
+    "title": "Virus Total APIV3",
+    "description": "user will be able to enter an ip address and get the exact experience of a report in virus total on this IP.",
+    "x-ms-api-annotation": {
+      "status": "Preview"
+    }
+  },
+  "host": "www.virustotal.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+
+  "paths": {
+    "/api/v3/ip_addresses/{ip}": {
+      "get": {
+        "summary": "Ip scan report V3",
+        "description": "scan Ip to get full report",
+        "operationId": "VirusTotalGetIpScanV3",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "ip",
+            "description": "Ip to verify",
```

Comment:
Can you change to `IP address` to match the description with the one in https://developers.virustotal.com/v3.0/reference#ip-info?

Reply:
done